osquery/tables/system/windows/logged_in_users.cpp

```diff
@@ -119,6 +119,31 @@ QueryData genLoggedInUsers(QueryContext& context) {
       r["host"] = std::string(addr, CLIENTADDRESS_LENGTH);
     }
 
+
+    if (!r["host"].length()) {
+      LPSTR clientName = nullptr;
+      res = WTSQuerySessionInformationA(WTS_CURRENT_SERVER_HANDLE,
```

Comment:
Why? r["host"] is not a wide char string anyway

Reply:
Because otherwise we do not support Unicode correctly.
We have conversion functions from UTF16 to UTF8 (what r["host"] would accept); we use `wstringToString`.